### ISSUE TITLE
Build of “aiy-vision-dkms” fails

### ISSUE THREAD

**Issue by haku15** [google/aiyprojects-raspbian#573]
I've posted this issue https://stackoverflow.com/questions/54450251/build-of-aiy-vision-dkms-fails but so far no answers. Does somebody have an idea? Maybe I am missing some preparations steps (kernel build?)?
Thanks for your help.

**Comment by dmitriykovalev** [comment]
Are you using the official Raspbian image from https://www.raspberrypi.org/downloads/raspbian/ for the build? Looks like `include/linux/iio/iio.h` is missing according to the error message you get: `error: unknown field ‘info_mask_separate’`.

**Comment by haku15** [comment]
Thanks Dmitriy for your help. Yes, it's an official Raspian image, but is not a fresh install, I've been using it for more than 1 year, and I've updated it in the last days.  Anyway, also thanks to your answer I've found the problem.
Under /lib/module I noticed there was an old 3.6 kernel called "3.6.trunk-rpi". The aiy installation was compiling also for this kernel. Suspecting that this would not work because of the too old kernel, I removed it with `sudo apt-get --purge remove  linux-headers-3.6-trunk-common`.
Then the aiy-vision-dkms  build was succesfull.
Thanks again
Sincerely
